scripts/run_tests.sh

```diff
@@ -39,6 +39,15 @@ function display_result {
 flake8 --max-complexity 10 --count
 display_result $? 1 "Code style check"
 
+pylint --rcfile=.pylintrc -j 0 ./app ./tests
+# pylint bit encodes the exit code to allow you to figure out which category has failed.
+# https://docs.pylint.org/en/1.6.0/run.html#exit-codes
+# http://stackoverflow.com/questions/6626351/how-to-extract-bits-from-return-code-number-in-bash
+# We want to fail if there are any errors or fatal errors so we use 3
+errorcode=$?
+(( res = $errorcode & 3 ))
+display_result $res "Code style check"
```

Comment:
It would be better to have a different message for the flake8 and pylint results that mentioned flake8/pylint